### PR TITLE
Add ability to name src blocks and auto-collapse code blocks.

### PR DESCRIPTION
**Pull request by rymndhng**
Solves #30

### CODE REVIEW

**Comment by stig** [comment]
Hello, thank you for your PR! I like it! (I'm afraid I totally missed the notification about this, so sorry for the wait.) That said, I've got a couple comments / questions:

1. Where did you find out about the `collapse` option? It's not listed here: https://jira.atlassian.com/secure/WikiRendererHelpAction.jspa?section=advanced -- is it available in all versions of JIRA?

2. It feels like you should be using `#+CAPTION:` rather than `#+NAME:` as per http://orgmode.org/manual/Images-and-tables.html#Images-and-tables

**Comment by rymndhng** [comment]
> Where did you find out about the collapse option? It's not listed here: https://jira.atlassian.com/secure/WikiRendererHelpAction.jspa?section=advanced -- is it available in all versions of JIRA?

I actually found this using the confluence code block macro (the doc site is down right now...). I did test it on JIRA and it worked as well.

> It feels like you should be using #+CAPTION: rather than #+NAME: as per http://orgmode.org/manual/Images-and-tables.html#Images-and-tables

Sounds right -- Will update the PR

**Comment by rymndhng** [comment]
@stig I've updated the PR. Not sure if I have the best implementation for building the caption text (it came back as a sequence of characters).

File: ox-jira.el [hunk 1]
```diff
@@ -40,6 +40,23 @@
 (require 'ox)
 (require 'ox-publish)
 
+
+;;; User Configurable Options
+(defgroup org-export-jira nil
```
Comment (stig) [inline ox-jira.el]: This need to be `ox-export-jira` I think. `org-jira` is a differet package :-)

File: ox-jira.el [hunk 2]
```diff
@@ -40,6 +40,23 @@
...

+;;; User Configurable Options
+(defgroup org-export-jira nil
+  "Options specific to JIRA export back-end."
+  :tag "Org Export JIRA"
+  :group 'org-export
+  :version "24.4"
+  :package-version '(ox-jira . "0.1"))
+
+(defcustom org-jira-src-collapse-threshold 30
```
Comment (stig) [inline ox-jira.el]: `ox-jira-src-collapse-threshold` please

File: ox-jira.el [hunk 3]
```diff
@@ -40,6 +40,23 @@
...
+  "Minimum number of lines in a src block to set collapse=true in JIRA/Confluence {code} block."
+  :group 'org-export-jira
```
Comment (stig) [inline ox-jira.el]: `ox-export-jira`

File: ox-jira.el [hunk 4]
```diff
@@ -96,6 +113,8 @@
     (verbatim . ox-jira-verbatim)
     (verse-block . (lambda (&rest args) (ox-jira--not-implemented 'verse-block))))
   :filters-alist '((:filter-parse-tree . ox-jira-fix-multi-paragraph-items))
+  :options-alist
+  '((:src-collapse-threshold nil nil org-jira-src-collapse-threshold))
```
Comment (stig) [inline ox-jira.el]: `ox-jira-src-...` please

**Review by stig — CHANGES_REQUESTED** [review]
Hello! This seems good to me, though please rename the variables  to use a `ox-jira` prefix rather than `org-jira` as org-jira refers to https://github.com/ahungry/org-jira rather than this project :-)

**Comment by rymndhng** [comment]
@stig I've updated the PR.

**Review by stig — APPROVED** [review]

**Comment by stig** [comment]
Thank you!

**Pull request merged**